Revert/Sources/AlertViewController.swift

```diff
@@ -49,10 +49,8 @@ extension AlertViewController {
     case .ActionSheet:
```

Comment:
Above here is some `@available(iOS 8.0, *)` directives.  These can probably be removed too?